migrations/versions/1471fe0d04ee_.py

```diff
@@ -0,0 +1,30 @@
+"""empty message
+
+Revision ID: 1471fe0d04ee
+Revises: 49f3a33f5437
+Create Date: 2018-06-08 19:32:47.485543
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = '1471fe0d04ee'
+down_revision = '49f3a33f5437'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('events', sa.Column('average_rating', sa.Float(), nullable=True))
+    op.add_column('events_version', sa.Column('average_rating', sa.Float(), autoincrement=False, nullable=True))
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column('events_version', 'average_rating')
+    op.drop_column('events', 'average_rating')
+    # ### end Alembic commands ###
```

Comment:
I don't think there is any need to store it as a field. Single source of truth principle states that information should not be replicated. We already have ratings stored in feedbacks, we don't need another field which will only reflect the redundant information and also be needed to be updated time and again

Reply:
@iamareebjamal If we do not store it, the average would be computed every time a GET request is made and can prove costly. What should be the alternative to storing this aggregate?

Reply:
This is how things are done generally where you have to get aggregate results. You can cache the results for some time, but that's it. It should never be stored as a field. Because it is not an implicit property of an event, but a calculated property

Reply:
Travis, GitHub, Twitter and every other API calculate things before displaying results, not every field in JSON maps to a field in database

Reply:
@iamareebjamal Cool, got it. I'll now remove the field and try to compute the aggregate for every request. I'll search, but do you know of another attribute on the server which is evaluated this way? So that I can write code in the same way?

Reply:
@schedutron I used `func.sum` but `func.avg` will do just fine here, or you could use plain Python and a `fields.Method`. Probably not the only way to do it but my attempts for the sales totals are [here](https://github.com/fossasia/open-event-server/pull/4821/files)

Reply:
@maxlorenz Thanks, but now I have to implement average rating without having an `average_rating` column corresponding in the db. I'm reading about how to do that.

Reply:
@maxlorenz Thanks a lot, having a method in the model with the `@property` decorator did the thing!